style.css

```diff
@@ -1259,14 +1259,36 @@ body.page .main-navigation {
 /*--------------------------------------------------------------
 ## Header
 --------------------------------------------------------------*/
-.site-header {
-  padding: 1em;
+
+.custom-header {
+    position: relative;
+    padding: 1rem;
+    display: block;
+    height: auto;
+}
+
+.has-header-image .custom-header .wp-custom-header img {
```

Comment:
Add the selector for header video.

Reply:
Which is probably:

`.has-header-media .custom-header .wp-custom-header video`